packages/api/src/models/alert.ts

```diff
@@ -99,7 +100,7 @@ const AlertSchema = new Schema<IAlert>(
     },
     team: {
       type: mongoose.Schema.Types.ObjectId,
-      ref: 'Team',
+      ref: Team.modelName,
```

Comment:
we should start to use this style to specify reference since the dependent model has to be imported first

ref: https://stackoverflow.com/questions/20832126/missingschemaerror-schema-hasnt-been-registered-for-model-user?newreg=660ccd5d84234efe83a6349aa6721460